universal-application-tool-0.0.1/app/views/LoginForm.java

```diff
@@ -1,11 +1,11 @@
 package views;
 
 import static com.google.common.base.Preconditions.checkNotNull;
-import static j2html.TagCreator.body;
-import static j2html.TagCreator.h1;
+import static j2html.TagCreator.*;
```

Comment:
don't use wildcards. intellij does this by default and you'll have to reconfigure it.

Reply:
I just learned this. There is a little button when you're editing a comment. It's the first button on the left of all the formatting buttons.

![Screen Shot 2021-02-23 at 5 21 56 PM](https://user-images.githubusercontent.com/5732310/108930938-a2975580-75fb-11eb-8ab7-eab1fe1fdced.png)

Reply:
Done!